.travis.yml

```diff
@@ -57,6 +57,9 @@ jobs:
     script:
       - export MPLBACKEND="AGG"
       - PATH=./linux/bin:$PATH py.test -n 1 --dist=loadfile --forked -v proteus/tests --ignore proteus/tests/POD -k 'not test_damBreak_solver_options' --cov=proteus
+      - git submodule update --init --recursive air-water-vv
+      - PATH=./linux/bin:$PATH py.test MPLBACKEND=Agg -n 1 --dist=loadfile --forked -v air-water-vv/Tests/1st_set
```

Comment:
I think the MPLBACKEND=Agg is confusing pytest.

Reply:
Yes I think that's the culprit as well. I just pushed the modification. Hopefully that's all it is.